<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bazaklubu</class>
 <widget class="QDialog" name="Bazaklubu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sec_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_usun">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tabela">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Załaduj tabele</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zawodnicy FCBarcelony:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>130</y>
     <width>91</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Imię</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nazwisko</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Wiek</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Wzrost</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Pozycja</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Numer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Lepsza noga</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>130</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_id"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_imie"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_nazwisko"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_wiek"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_wzrost"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pozycja"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_numer"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_lnoga"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
